src/snowflake/connector/__init__.py

```diff
@@ -9,6 +9,7 @@
 apilevel = "2.0"
 threadsafety = 2
 paramstyle = "pyformat"
+log_imported_packages_in_telemetry = True
```

Comment:
Let's please use a connection parameter instead of this global variable

Reply:
done